src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/LeggTilEllerFjernBrevmottakere/useLeggTilFjernBrevmottaker.ts

```diff
@@ -45,14 +50,43 @@ export interface IRestBrevmottaker {
 
 const useLeggTilFjernBrevmottaker = () => {
     const { settToast } = useApp();
-    const { settÅpenBehandling } = useBehandling();
+    const { åpenBehandling: åpenBehandlingRessurs, settÅpenBehandling } = useBehandling();
     const { behandlingId } = useSakOgBehandlingParams();
     const { request } = useHttp();
+    const åpenBehandling = hentDataFraRessurs(åpenBehandlingRessurs);
 
     const mottaker = useFelt<Mottaker | ''>({
         verdi: '',
-        valideringsfunksjon: felt =>
-            felt.verdi !== '' ? ok(felt) : feil(felt, 'Feltet er påkrevd'),
+        valideringsfunksjon: felt => {
+            if (felt.verdi === '') {
+                return feil(felt, 'Feltet er påkrevd');
+            }
+            if (åpenBehandling?.brevmottakere.length) {
+                const eksisterendeBrevmottakerType = åpenBehandling.brevmottakere[0].type;
+                if (felt.verdi === eksisterendeBrevmottakerType) {
+                    return feil(felt, `${mottakerVisningsnavn[felt.verdi]} er allerede lagt til`);
+                }
+                if (
+                    felt.verdi === Mottaker.DØDSBO ||
+                    eksisterendeBrevmottakerType === Mottaker.DØDSBO
+                ) {
+                    return feil(felt, 'Dødsbo kan ikke kombineres med andre brevmottakere');
+                }
+                if (
+                    felt.verdi === Mottaker.BRUKER_MED_UTENLANDSK_ADRESSE ||
+                    eksisterendeBrevmottakerType === Mottaker.BRUKER_MED_UTENLANDSK_ADRESSE
+                ) {
+                    return ok(felt);
+                }
+                return feil(
+                    felt,
+                    `${
+                        mottakerVisningsnavn[eksisterendeBrevmottakerType]
+                    } kan ikke kombineres med ${mottakerVisningsnavn[felt.verdi].toLowerCase()}.`
+                );
```

Comment:
Kunne kanskje hatt en mer eksplisitt sjekk iht. spec her også, f.eks noe sånt:
```suggestion
                if (
                    felt.verdi in [Mottaker.FULLMEKTIG, Mottaker.VERGE] &&
                    eksisterendeBrevmottakerType in [Mottaker.FULLMEKTIG, Mottaker.VERGE]
                ) {
                    return feil(
                        felt,
                        `${
                            mottakerVisningsnavn[eksisterendeBrevmottakerType]
                        } kan ikke kombineres med ${mottakerVisningsnavn[
                            felt.verdi
                        ].toLowerCase()}.`
                    );
                }
```
Blir bra uansett :)

Reply:
Har tenkt på det, og er litt enig, men samtidig så fungerer dette også som en catch-all-feil som er logisk korrekt. Resultat `ok` er kun dersom man ikke har tidligere lagrede mottakere eller hvis man treffer forrige if (kun én av verdiene er utenlandsk adresse). Og denne feilmeldingen passer da til tilfeller som ikke er dødsbo eller samme mottakertype som eksisterende 🤔

Reply:
Gå for det du er mest komfortabel med, men trodde logisk sett det skulle være det samme, siden samme mottakertype som eksisterende blir fanget opp av den første if'en 🤔

Reply:
hvis jeg legger den siste inn i en `if (en fullmektig og en verge)` så må jeg uansett også ha en `else return feil(felt, 'ugyldig kombinasjon')` 🤷

Reply:
Ja, hvis du vil sikre det mot andre typer mottakere som kan komme senere, så må du nok uansett ha en ekstra else :)